Geometry/CMSCommonData/python/cmsExtendedGeometry2017devXML_cfi.py

```diff
@@ -0,0 +1,291 @@
+        'Geometry/TrackerCommonData/data/tecpetal0f.xml',
+        'Geometry/TrackerCommonData/data/tecpetal0b.xml',
+        'Geometry/TrackerCommonData/data/tecpetal3.xml',
+        'Geometry/TrackerCommonData/data/tecpetal3f.xml',
+        'Geometry/TrackerCommonData/data/tecpetal3b.xml',
+        'Geometry/TrackerCommonData/data/tecpetal6f.xml',
+        'Geometry/TrackerCommonData/data/tecpetal6b.xml',
+        'Geometry/TrackerCommonData/data/tecpetal8f.xml',
+        'Geometry/TrackerCommonData/data/tecpetal8b.xml',
+        'Geometry/TrackerCommonData/data/tecwheel.xml',
+        'Geometry/TrackerCommonData/data/tecwheela.xml',
+        'Geometry/TrackerCommonData/data/tecwheelb.xml',
+        'Geometry/TrackerCommonData/data/tecwheelc.xml',
+        'Geometry/TrackerCommonData/data/tecwheeld.xml',
+        'Geometry/TrackerCommonData/data/tecwheel6.xml',
+        'Geometry/TrackerCommonData/data/tecservices.xml',
+        'Geometry/TrackerCommonData/data/tecbackplate.xml',
+        'Geometry/TrackerCommonData/data/tec.xml',
+        'Geometry/TrackerCommonData/data/Run2/trackermaterial.xml',
+        'Geometry/TrackerCommonData/data/Run2/tracker.xml',
+        'Geometry/TrackerCommonData/data/trackerpixbar.xml',
+        'Geometry/TrackerCommonData/data/PhaseI/trackerpixfwd.xml',
+        'Geometry/TrackerCommonData/data/trackertibtidservices.xml',
+        'Geometry/TrackerCommonData/data/trackertib.xml',
+        'Geometry/TrackerCommonData/data/trackertid.xml',
+        'Geometry/TrackerCommonData/data/trackertob.xml',
+        'Geometry/TrackerCommonData/data/trackertec.xml',
+        'Geometry/TrackerCommonData/data/v2/trackerbulkhead.xml',
+        'Geometry/TrackerCommonData/data/trackerother.xml',
+        'Geometry/EcalCommonData/data/Run2/eregalgo.xml',
+        'Geometry/EcalCommonData/data/ebalgo.xml',
+        'Geometry/EcalCommonData/data/ebcon.xml',
+        'Geometry/EcalCommonData/data/ebrot.xml',
+        'Geometry/EcalCommonData/data/eecon.xml',
+        'Geometry/EcalCommonData/data/eefixed.xml',
+        'Geometry/EcalCommonData/data/eehier.xml',
+        'Geometry/EcalCommonData/data/eealgo.xml',
+        'Geometry/EcalCommonData/data/escon.xml',
+        'Geometry/EcalCommonData/data/esalgo.xml',
+        'Geometry/EcalCommonData/data/eeF.xml',
+        'Geometry/EcalCommonData/data/eeB.xml',
+        'Geometry/EcalCommonData/data/ectkcable.xml',
+        'Geometry/HcalCommonData/data/hcalrotations.xml',
+        'Geometry/HcalCommonData/data/PhaseI/hcalalgo.xml',
+        'Geometry/HcalCommonData/data/hcalcablealgo.xml',
+        'Geometry/HcalCommonData/data/hcalbarrelalgo.xml',
+        'Geometry/HcalCommonData/data/PhaseI/hcalendcapalgo.xml',
+        'Geometry/HcalCommonData/data/hcalouteralgo.xml',
+        'Geometry/HcalCommonData/data/hcalforwardalgo.xml',
+        'Geometry/HcalCommonData/data/average/hcalforwardmaterial.xml',
+        'Geometry/HcalCommonData/data/Run2/hcalSimNumbering17Plan0.xml', 
```

Comment:
@kpedro88 - indeed. The scenario will not have a relval workflow nor a usual set of geometry configuration files. It will be removed after the cavern is integrated in to a 2023 scenario. See https://github.com/cms-sw/cmssw/pull/17831

Our primary concern for the moment is overlaps and description performance - the shapes are more complex. In the meantime it is used for tests with --geometry option which takes a file name and a specific --GT